sdk/storage/Azure.Storage.Files.Shares/src/Generated/FileRestClient.cs

```diff
@@ -11519,10 +11519,10 @@ internal ShareProtocolSettings(bool skipInitialization)
         /// Serialize a ShareProtocolSettings instance as XML.
         /// </summary>
         /// <param name="value">The ShareProtocolSettings instance to serialize.</param>
-        /// <param name="name">An optional name to use for the root element instead of "ShareProtocolSettings".</param>
+        /// <param name="name">An optional name to use for the root element instead of "ProtocolSettings".</param>
         /// <param name="ns">An optional namespace to use for the root element instead of "".</param>
         /// <returns>The serialized XML element.</returns>
-        internal static System.Xml.Linq.XElement ToXml(Azure.Storage.Files.Shares.Models.ShareProtocolSettings value, string name = "ShareProtocolSettings", string ns = "")
+        internal static System.Xml.Linq.XElement ToXml(Azure.Storage.Files.Shares.Models.ShareProtocolSettings value, string name = "ProtocolSettings", string ns = "")
```

Comment:
we should add test coverage for this (and the other change).

Reply:
I should add another transform to switch this back actually

Reply:
Nevermind, I'm wrong it should be this.